app/javascript/grid.js

```diff
@@ -51,7 +86,23 @@ export class BlockGrid {
     }
 
     blockClicked (e, block) {
-        console.log(e, block);
+        
+	let cut_me = e.target.id
+	let x_in = parseInt (cut_me.substring(6, 7), 10);//Get x parameter
+	let y_in = parseInt (cut_me.substring(8, 9), 10);//Get y parameter
+	let c_in = document.getElementById(e.target.id).style.backgroundColor//Get colour parameter
```

Comment:
why **let** not **const**?

Reply:
Did some reading on ES6, will now prioritize const where possible. Fixed in last commit.